examples/with-vue-thirdpartyemailpassword/src/router/index.ts

```diff
@@ -0,0 +1,29 @@
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import HomeView from '../views/HomeView.vue'
+
+Vue.use(VueRouter)
+
+const router = new VueRouter({
+  mode: 'history',
+  base: import.meta.env.BASE_URL,
+  routes: [
+    {
+      path: '/',
+      name: 'home',
+      component: HomeView
+    },
+    {
+      path:'/auth*',
+      name:'auth',
+      component: ()=> import('../views/AuthView.vue'),
+    },
+  
+    
+   
+    
+   
```

Comment:
you can remove the additional newlines

Reply:
Don't see these new lines in my editor?

Reply:
hmm that's strange, git is picking up the new lines though